src/test/java/zaasclient/HttpsClientTest.java

```
@@ -0,0 +1,86 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package zaasclient;
+
+import org.apache.http.impl.client.BasicCookieStore;
+import org.apache.http.impl.cookie.BasicClientCookie;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+import java.util.Properties;
+
+import static org.junit.Assert.assertNotNull;
+
+public class HttpsClientTest {
+    private static final String CONFIG_FILE_PATH = "src/test/resources/configFile.properties";
+    private ConfigProperties configProperties;
+    private HttpsClient httpsClient;
+
+    @Before
+    public void setup() throws IOException {
+        this.configProperties = getConfigProperties();
+        this.httpsClient = new HttpsClient(configProperties);
+    }
+
+    private ConfigProperties getConfigProperties() throws IOException {
+        String absoluteFilePath = new File(CONFIG_FILE_PATH).getAbsolutePath();
+        ConfigProperties properties = new ConfigProperties();
+        Properties configProp = new Properties();
+        try {
+            if (Paths.get(absoluteFilePath).toFile().exists()) {
```

what happens if file does not exist ? Do we proceed as if it is OK ? I guess we should fail fast throwing the error message that file does not exist